### DISCUSSION TITLE
OpenDTU Compile Hilfe für Fork gesucht

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#2196] [General]
Hi,
ich würde mich mal wieder gern an das selber kompilieren heranwagen, ist schon eine Weile her. Das kompilieren an sich hat ja immer geklappt.
Der aktuelle Stand ist VSCode ist installiert und die Wiki Seite zum „Compile OpenDTU“ ist gelesen, Github ist installiert.

Edit: Beitrag angepasst

**Comment by Tiese** [comment]
Hallo Thorsten,
ich habe mir inzwischen das [hier](https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/1108) gebaut. &#128515;
Seit dem spiele ich nur noch die Originalversionen auf - fertig. &#128521;

**Comment by jblond4711** [comment]
Hi,
alles gut und schön, hilft mir nur leider nicht so wirklich weiter.

**Reply by Tiese** [reply]
Ja - und selbst mit meiner Hilfe hat es letztes Mal auch nicht geklappt!

**Reply by jblond4711** [reply]
Das kannst du doch garnicht vergleichen da es ja damals um was ganz anderes ging.

**Reply by Tiese** [reply]
Oh, bitte entschuldige, dass ich dir jemals geholfen habe - wird bestimmt nicht wieder vorkommen!
